tests/testthat/test-index-fun.R

```diff
@@ -81,6 +81,9 @@ test_that("some S3 methods for yearweek, yearmonth & yearquarter", {
   expect_is(unique(y2), "yearquarter")
   expect_identical(yearquarter(y), y)
   expect_is(y[1:2], "yearquarter")
+  expect_equal(yearquarter("2010 Q1"), y[1])
```

Comment:
Can you make a new test block to include the tests, and refer to the issue in the `test_that(..., {})`?

Reply:
Fixed.